desktop/src/main/java/bisq/desktop/components/PeerInfoIcon.java

```diff
@@ -146,19 +143,19 @@ private PeerInfoIcon(NodeAddress nodeAddress,
 
         boolean isFiatCurrency = CurrencyUtil.isFiatCurrency(offer.getCurrencyCode());
 
-        String accountAge = isFiatCurrency ?
-                peersAccount.first > -1 ? Res.get("peerInfoIcon.tooltip.age", DisplayUtils.formatAccountAge(peersAccount.first)) :
+        String accountAgeTooltip = isFiatCurrency ?
+                accountAge > -1 ? Res.get("peerInfoIcon.tooltip.age", DisplayUtils.formatAccountAge(accountAge)) :
                         Res.get("peerInfoIcon.tooltip.unknownAge") :
                 "";
         tooltipText = hasTraded ?
-                Res.get("peerInfoIcon.tooltip.trade.traded", role, fullAddress, numTrades, accountAge) :
-                Res.get("peerInfoIcon.tooltip.trade.notTraded", role, fullAddress, accountAge);
+                Res.get("peerInfoIcon.tooltip.trade.traded", role, fullAddress, numTrades, accountAgeTooltip) :
+                Res.get("peerInfoIcon.tooltip.trade.notTraded", role, fullAddress, accountAgeTooltip);
 
         // outer circle
         Color ringColor;
         if (isFiatCurrency) {
 
-            switch (accountAgeWitnessService.getPeersAccountAgeCategory(peersAccount.first)) {
+            switch (accountAgeWitnessService.getPeersAccountAgeCategory(accountAge)) {
```

Comment:
I think this should switch on `accountAge` for non signed accounts and switch on `signAge` for signed accounts.

Possibly it should switch on `signAge` for all accounts of risky payment methods. Otherwise it will show that an older account that isn't signed is safe, even though it's a risky method.

Reply:
Yes, I wasn't sure in that case if we should use always accountAge or selectively switch the colors. But true for people used to the color codes it might trick them into false security if always accountAge is used. Will adapt it based on chargeback risk.